src/App.js

```diff
@@ -1,5 +1,9 @@
 import logo from './logo.svg';
 import './App.css';
+import store from './Redux/store';
+import { getExperiencesData } from './Redux/ExperiencesSlice/ExperiencesSlice';
+
+store.dispatch(getExperiencesData());
```

Comment:
@HFG43 I kindly suggest you use the `useDispatch` hook to communicate with the store and execute the specified action. While your current code works, importing the store into a component links that component to that exact store instance, and it already has the store initialized in `index.js`. This will make the code more difficult to test and the component impossible to reuse correctly. You can call each action in its corresponding component or you can create a custom hook where you handle all calls to the store with `useDispatch`, cause this hook implementation just returns the `store.dispatch`